frontend/public/components/terminal.jsx

```diff
@@ -3,16 +3,20 @@ import ReactDOM from 'react-dom';
 import PropTypes from 'prop-types';
 import { Terminal as XTerminal } from 'xterm';
 import * as fit from 'xterm/lib/addons/fit/fit';
+import * as full from 'xterm/lib/addons/fullscreen/fullscreen';
 
 XTerminal.applyAddon(fit);
+XTerminal.applyAddon(full);
 
 export class Terminal extends React.Component {
   constructor(props) {
     super(props);
     this.state = {height: 0, width: 0};
     this.innerRef = React.createRef();
     this.outerRef = React.createRef();
+    this.isFullscreen = false;
     this.onResize = () => this.onResize_();
+    this.setFullscreen = fullscreen => this.setFullscreen_(fullscreen);
```

Comment:
same comment: not needed with fat arrow in onClick

Reply:
fixed